tasks/setup-FreeBSD.yml

```diff
@@ -0,0 +1,3 @@
+---
+- name: Install Varnish.
+  pkgng: name=varnish4 state=present
```

Comment:
For other OSes, the version is configurable (3.x, 4.x, 5.x, etc.). Is it possible to make this easily configurable for FreeBSD too?